### PR TITLE
Assign alert profile to the enterprise when deploying ManageIQ

### PR DESCRIPTION
**Pull request by elad661**
This still uses the hack from my pending PR to manageiq-api.

It will work for now, but a more "proper" solution (until there's actual API for this) would be to use `oc rsh` to run some ruby code to make that assignment.

cc @bazulay

### CODE REVIEW

**Comment by elad661** [comment]
cc @moolitayer 

This now uses `oc rsh` and the rails runner, which makes it work even if we drop our custom patch that adds API for profile assignment.

**Comment by bazulay** [comment]
I didn't check the script but the approach looks good.
We will need to push forward the additional API , but I agree that until it is solved we can use this patch.
so for this stage this patch LGTM

File: deployer.sh [hunk 1]
```diff
@@ -220,6 +221,15 @@ else
       export OPENSHIFT_CA_CRT="$(${SSH_COMMAND} cat /etc/origin/master/ca.crt)"
       export OPENSHIFT_MANAGEMENT_ADMIN_TOKEN="$(${SSH_COMMAND} oc sa get-token -n management-infra management-admin)"
 
+      echo "Assigning alert profiles to Enterprise in ManageIQ"
+      cat <<EOF > profile_assign.sh
+      source /etc/default/evm
+      rails r 'MiqAlertSet.find_by(:guid=>"a16fcf51-e2ae-492d-af37-19de881476ad").assign_to_objects(MiqEnterprise.last); MiqAlertSet.find_by(:guid=>"ff0fb114-be03-4685-bebb-b6ae8f13d7ad").assign_to_objects(MiqEnterprise.last)'
```
Comment (moolitayer) [inline deployer.sh]: it would be nice to extract the ruby code to a file of all ruby code executed in the manageiq container

**Comment by moolitayer** [comment]
Implementation looks good

**Comment by elad661** [comment]
I extracted the ruby code and the shell script we run on the container to separate files. This will make it easier to run more code there in the future, for example enabling C&U roles or doing basic sanity tests.

**Review by moolitayer — APPROVED** [review]
LGTM :+1:

**Pull request merged**